src/main/java/lotto/domain/Lottos.java

```diff
@@ -0,0 +1,56 @@
+package lotto.domain;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Objects;
+
+public class Lottos {
+
+    private final List<Lotto> lottos;
+
+    private Lottos(List<Lotto> lottos) {
+        this.lottos = lottos;
```

Comment:
Collection에서 제공하는 unmodifiableList를 활용해보는 건 어떨까요?
리스트에 대한 추가, 삭제를 막아주는 기능으로 일급 컬렉션도 좀 더 불변 객체에 가까워질 수 있습니다 :)

```suggestion
        this.lottos = Collections.unmodifiableList(lottos);
```

Reply:
좋은 방법인 것 같습니다!!